sdk/formrecognizer/azure-ai-formrecognizer/pom.xml

```diff
@@ -36,25 +36,25 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.21.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.22.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
This should have been updated when the auto-increment PR was created after core release.

Reply:
Oh, I see that this is merging into a release branch. Why are we not using `main` to release this?

Reply:
yeah,  this is since we reset the head on the respective sdk folder root for thecreating the service branch release.
`git reset HEAD /sdk/formrecognizer/ `

Reply:
Since, we have ongoing `beta` development on the main branch.